vars/buildDockerAndPublishImage.groovy

```diff
@@ -9,19 +9,23 @@ def call(String imageShortName, Map userConfig=[:]) {
     automaticSemanticVersioning: false, // Do not automagically increase semantic version by default
     includeImageNameInTag: false, // Set to true for multiple semversioned images built in parallel, will include the image name in tag to avoid conflict
     dockerfile: 'Dockerfile', // Obvious default
-    platform: 'linux/amd64', // Intel/AMD 64 Bits, following Docker platform identifiers
+    platform: '', // Kept for backward compatibility to be deprecated
```

Comment:
```suggestion
```

You can remove it (and use Groovy-isms to test for emptiness or nulliness such as https://docs.oracle.com/en/cloud/saas/applications-common/23b/cgsac/understanding-null-versus-the-empty-string.html#u30004090 or https://www.tutorialspoint.com/groovy/groovy_containskey.htm)